src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/search.html.twig

```diff
@@ -38,6 +40,7 @@
                                         data-sftype="{{ secondFactor.type }}"
                                         data-sfidentityid="{{ secondFactor.identityId }}"
                                         data-sfname="{{ secondFactor.name }}"
+                                        data-sfdocumentnumber="{{ secondFactor.documentNumber }}"
```

Comment:
These data attributes are used for displaying a token revocation confirmation modal. If this is your intention, please [add it to the JavaScript](https://github.com/SURFnet/Stepup-RA/blob/02038b99b6406faea1b1b5f318349f8f2eedf870/src/Surfnet/StepupRa/RaBundle/Resources/public/js/stepup-ra.js#L23-L36). If not, there's no need to add this.